### PR TITLE
process component_verify_ticket and unauthorized message

### PR DESCRIPTION
**Pull request by hunter007**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: 9243b1e00ce3880762486997e9686c7088ff7e01
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: a03a890c6282908148c6b76d03b6a7b658cc4971
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: 9243b1e00ce3880762486997e9686c7088ff7e01
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: a03a890c6282908148c6b76d03b6a7b658cc4971
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: fc1bd91ad7c25dcf2d7997e6ac13b6de56b3b71e
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

File: wechatpy/component.py [hunk 1]
```diff
@@ -9,17 +9,68 @@
...
 import requests
+import xmltodict
 
 from wechatpy._compat import get_querystring
+from wechatpy.utils import to_text, to_binary, timezone
+from wechatpy.fields import IntegerField, StringField
+from wechatpy.messages import MessageMetaClass
 from wechatpy.session.memorystorage import MemoryStorage
 from wechatpy.exceptions import WeChatClientException, APILimitedException
+from wechatpy.enterprise.crypto import WeChatCrypto
```
Comment (messense) [inline wechatpy/component.py]: 这里用 enterprise 的 crypto 不太合适吧？继承一个新的出来用吧。

Comment (hunter007) [inline wechatpy/component.py]: 写错了，我本意是想用 from wechatpy.crypto import WeChatCrypto

因为加密算法都是一样的，就不再添加多余代码。

# 

赵文涛  steven zhao

Tel：（+86） 150 051 72386

# Email：wentao79@gmail.com

2015-09-25 13:34 GMT+08:00 messense notifications@github.com:

Comment (messense) [inline wechatpy/component.py]: 嗯，用那个也可以，稍微改一下吧。Thanks.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: 9243b1e00ce3880762486997e9686c7088ff7e01
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: a03a890c6282908148c6b76d03b6a7b658cc4971
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: fc1bd91ad7c25dcf2d7997e6ac13b6de56b3b71e
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: 730b11b2ca36d88dc591854e199afcc9e2fa377e
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

**Comment by messense** [comment]
Closes #50

**Pull request merged**